Build failure with fmt 8.0.0

**Version, Operative system**
0.99.30. Have not tried the latest version but the code hasn't changed

**Describe the bug**
cherrytree fails to compile with latest fmt: https://github.com/fmtlib/fmt/releases/tag/8.0.0

```
In file included from /usr/include/spdlog/fmt/fmt.h:25,
                 from /usr/include/spdlog/common.h:36,
                 from /usr/include/spdlog/spdlog.h:12,
                 from /builds/z3ntu/aports/testing/cherrytree/src/cherrytree_0.99.30/src/ct/ct_logging.h:26,
                 from /builds/z3ntu/aports/testing/cherrytree/src/cherrytree_0.99.30/src/ct/ct_misc_utils.h:27,
                 from /builds/z3ntu/aports/testing/cherrytree/src/cherrytree_0.99.30/src/ct/ct_main_win.h:33,
                 from /builds/z3ntu/aports/testing/cherrytree/src/cherrytree_0.99.30/src/ct/ct_actions.h:26,
                 from /builds/z3ntu/aports/testing/cherrytree/src/cherrytree_0.99.30/src/ct/ct_actions_others.cc:24:
/usr/include/fmt/core.h: In instantiation of 'constexpr fmt::v7::detail::value<Context> fmt::v7::detail::make_arg(const T&) [with bool IS_PACKED = true; Context = fmt::v7::basic_printf_context<fmt::v7::appender, char>; fmt::v7::detail::type <anonymous> = fmt::v7::detail::type::custom_type; T = Glib::ustring; typename std::enable_if<IS_PACKED, int>::type <anonymous> = 0]':
/usr/include/fmt/core.h:1705:64:   required from 'constexpr fmt::v7::format_arg_store<Context, Args>::format_arg_store(const Args& ...) [with Context = fmt::v7::basic_printf_context<fmt::v7::appender, char>; Args = {Glib::ustring}]'
/usr/include/fmt/core.h:1721:18:   required from 'constexpr fmt::v7::format_arg_store<Context, Args ...> fmt::v7::make_format_args(const Args& ...) [with Context = fmt::v7::basic_printf_context<fmt::v7::appender, char>; Args = {Glib::ustring}]'
/usr/include/fmt/printf.h:576:70:   required from 'std::__cxx11::basic_string<Char> fmt::v7::sprintf(const S&, const T& ...) [with S = std::__cxx11::basic_string<char>; T = {Glib::ustring}; Char = char]'
/builds/z3ntu/aports/testing/cherrytree/src/cherrytree_0.99.30/src/ct/ct_actions_others.cc:287:105:   required from here
/usr/include/fmt/core.h:1579:7: error: static assertion failed: Cannot format an argument. To make type T formattable provide a formatter<T> specialization: https://fmt.dev/latest/api.html#udt
 1579 |       !std::is_same<decltype(arg), const unformattable&>::value,
      |       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Relevant line in source: https://github.com/giuspen/cherrytree/blob/0.99.30/src/ct/ct_actions_others.cc#L287

--- a/src/ct/ct_actions_others.cc
+++ b/src/ct/ct_actions_others.cc
@@ -289,7 +289,7 @@ void CtActions::link_clicked(const Glib::ustring& tag_property_value, bool from_
     if (link_entry.type == CtConst::LINK_TYPE_WEBS) { // link to webpage
         Glib::ustring clean_weblink = str::replace(link_entry.webs, "amp;", "");
         if (_pCtMainWin->get_ct_config()->weblinkCustomOn) {
-            std::string cmd = fmt::sprintf(_pCtMainWin->get_ct_config()->weblinkCustomAct, clean_weblink);
+            std::string cmd = fmt::sprintf(_pCtMainWin->get_ct_config()->weblinkCustomAct, clean_weblink.raw());
             int retr = std::system(cmd.c_str());
             if (retr == -1) {
                 // Internal std::system error

